shim/opentelemetry-opentracing-shim/src/opentelemetry/shim/opentracing_shim/__init__.py

```diff
@@ -280,7 +295,16 @@ def log_kv(
             event_timestamp = None
 
         event_name = util.event_name_from_kv(key_values)
-        self._otel_span.add_event(event_name, key_values, event_timestamp)
+
+        # Sanitize the key_values dictionary
+        sanitized_key_values = {
+            k: self._sanitize_attribute_value(k, v)
+            for k, v in key_values.items()
+        }
+
+        self._otel_span.add_event(
```

Comment:
Since `add_event` uses [BoundedAttributes](https://github.com/open-telemetry/opentelemetry-python/blob/main/opentelemetry-sdk/src/opentelemetry/sdk/trace/__init__.py#L871) aren't all values already [sanitized](https://github.com/open-telemetry/opentelemetry-python/blob/main/opentelemetry-api/src/opentelemetry/attributes/__init__.py#L176) to conform to the OpenTelemetry `AttributeValue` definition?

Reply:
Indeed, but if we don't have these code, the value will not be reformated to str.

Should I do some change in [BoundedAttributes](https://github.com/open-telemetry/opentelemetry-python/blob/main/opentelemetry-sdk/src/opentelemetry/sdk/trace/__init__.py#L871)

Reply:
@zhihali do you mean change _clean_attribute to cast everything else to str?